pom.xml

```diff
@@ -207,6 +207,11 @@
         </exclusions>
       </dependency>
       <dependency>
+        <groupId>com.google.protobuf</groupId>
```

Comment:
does this dependency need to be in the top-level pom file or can we push it to the servers module?